spec/server/server.spec.js

```diff
@@ -2,15 +2,10 @@ import request from 'supertest';
 
 import server from '../../server/server';
 
-describe('App Server', () => {
+describe('App Server', () => { 
```

Comment:
This test is necessary with how the server catches all things to client side render. Perhaps a test to ensure we're getting back static assets instead would be more meaningful

Reply:
Cool - sounds like a reasonable test.  Thanks for the suggestion, will make the update.